### ISSUE TITLE
I Wish onRefresh Was Optional

### ISSUE THREAD

**Issue by dragos-rosca** [osstotalsoft/rocket-webapp-components#10]
https://github.com/osstotalsoft/rocket-webapp-components/blob/27d56030e965011ef675f5cc067c23085b5b09ad/components/Pagination/Pagination.js#L61

I wish I could send undefined to "onRefresh" and have a option for that icon to not show on screen.

As a suggestion we could check to see if we should draw the button
```javascript
{ !!onRefresh && <IconButton onClick={handleRefresh} color="defaultNoBackground">
          <RefreshIcon />
        </IconButton >
}
```

**Comment by VCuzmin** [comment]
Will be fixed in the upcoming release!

**Comment by VCuzmin** [comment]
Fixed in **v0.0.16** release!
